hack/setup-e2e.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
not need this one, the CI will create the namespace for us

Reply:
But how do we passing the ns created by the CI to the test itself?